### PR TITLE
Adding an acceptance test against preint after a successful master build  FIT-4

### PR DESCRIPTION
**Pull request by edgibbs**

### CODE REVIEW

**Comment by edgibbs** [comment]
retest this please

**Comment by edgibbs** [comment]
retest this please

**Comment by edgibbs** [comment]
retest this please

**Comment by edgibbs** [comment]
retest this please

**Comment by edgibbs** [comment]
retest this please

**Comment by edgibbs** [comment]
retest this please

**Comment by edgibbs** [comment]
retest this please

**Comment by edgibbs** [comment]
retest this please

**Comment by edgibbs** [comment]
retest this please

**Comment by edgibbs** [comment]
retest this please

File: spec/acceptance/landing_page_spec.rb [hunk 1]
```diff
@@ -1,16 +1,27 @@
 # frozen_string_literal: true
 
 require 'acceptance_helper'
+require 'feature'
 
 feature 'Landing Page' do
-  scenario 'has a dashboard' do
-    visit '/?token=example'
+  scenario 'has some cases and referrals' do
+    login
```
Comment (denys-davydov) [inline spec/acceptance/landing_page_spec.rb]: 👍 👍 👍

**Pull request merged**